sparkle/src/components/Tab.tsx

```diff
@@ -63,7 +64,7 @@ export function Tab({ tabs, onTabClick, className = "" }: TabProps) {
         : iconClasses.default;
 
       const finalTabClasses = classNames(
-        "flex gap-x-2 text-sm px-4 py-3 border-b-2 transition-colors duration-200 whitespace-nowrap",
+        "group flex gap-x-2 text-sm font-medium px-4 py-3 border-b-2 transition-colors duration-200 whitespace-nowrap",
```

Comment:
based on your Figma it should be pl-3 pr-4 instead of px-4 right? Just checking I get this right :)

Reply:
I'm doing back and forth between figma and code see what works and don't.
I tried to visually align the icon, but it breaks a bucnh of stuff, went back to a more simple px-4

Reply:
Roger that :+1: